README.md

```diff
@@ -24,9 +24,12 @@ Use conditional statements as expressions, instead of awkward nested ternaries:
 
 ```js
 let x = do {
-  if (foo()) { f() }
-  else if (bar()) { g() }
-  else { h() }
+  if (foo())
```

Comment:
It's a very standard practice in JS (and other languages with optional curly-braces) to just _always_ use them.

Reply:
The curly braces aren't as much the problem as much as putting both the condition and the statement on the same line, separating them was just to improve readability.